client/main.lua

```diff
@@ -129,7 +129,7 @@ end)
 RegisterNetEvent('prison:client:Leave')
 AddEventHandler('prison:client:Leave', function()
 	if jailTime > 0 then 
-		QBCore.Functions.Notify("You still have to... "..jailTime.." months..")
+		QBCore.Functions.Notify("You still have "..jailTime.." months left in your sentence")
```

Comment:
"Better grammar" will be: `QBCore.Functions.Notify(jailTime.." months left to sentence")`

The player already knows that the notification is for them, so `You still have it` is not necessary